projects/demo-playwright/tests/deep/deep.ts

```diff
@@ -0,0 +1,74 @@
+import {Locator, Page, test} from '@playwright/test';
+
+export class DeepSpecHelpers {
```

Comment:
Should we transform it to `TuiDocumentationAPIPagePO`?

```ts
const apiPage = new TuiDocumentationAPIPagePO(page);
```

Reply:
fixed